src/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Outlandish\Wpackagist\EventListener;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+
+class ExceptionListener
+{
+    public function onKernelException(ExceptionEvent $event)
+    {
+        // Let Symfony's default error tracing happen in dev.
+        if (getenv('APP_ENV') === 'dev') {
```

Comment:
This isn't working locally. Is it working for you?

Reply:
@rasmuswinter it works for me via Docker. If you're running natively you might need to set `APP_ENV` in the vhost / wherever you configure the server process? Maybe there's a better way, given Symfony should default to `dev` internally, but I'm not sure what.

Reply:
I think the issue is that `dotenv` is initialised with `$usePutEnv=false`, so the variables in the .env files shouldn't then be stored as system environment variables. Not sure why docker would be different. Changing this to `if($_SERVER['APP_ENV'] === 'dev')` instead does work

Reply:
Thanks @rasmuswinter, it looks like they have a reason to default that flag off, and that superglobal is how Symfony boilerplate uses this var, so let's go with that.